src/main/java/ttt/LineGenerator.java

```diff
@@ -0,0 +1,55 @@
+package ttt;
+
+public class LineGenerator {
+    private final Cell[] grid;
+
+    public LineGenerator(Cell[] grid) {
+        this.grid = grid;
+    }
+
+    public Cell[][] linesForAllDirections() {
+        Cell[][] allLines = new Cell[8][];
+        allLines[0] = topRow();
+        allLines[1] = middleRow();
+        allLines[2] = bottomRow();
+        allLines[3] = leftColumn();
+        allLines[4] = middleColumn();
+        allLines[5] = rightColumn();
+        allLines[6] = backslashDiagonal();
+        allLines[7] = forwardslashDiagonal();
+
+        return allLines;
+    }
```

Comment:
When I first saw this I thought that the reason for you to put the elements so ordered in the `Cell[][]` was because there is a level of "connascence" (https://en.wikipedia.org/wiki/Connascence_(computer_programming)) of the ordering of the rows and columns outside of the class. Thankfully this isn't so! :+1: 

Given that this is not so, you could simplify the code like so:

```
public Cell[][] linesForAllDirections() {
        return new Cell[][]{
            topRow(), middleRow(), bottomRow(), leftColumn(),
            middleColumn(), rightColumn(), backslashDiagonal(),
            forwardslashDiagonal()
        };
    }
```

The reason being that we don't need that variable `allLines` other that for the creation of the `Cell[][]` object. Also, given of the clean naming of the methods that make up the rows and columns there is no apparent loss of readability.